### PR TITLE
[indexer-alt] add cp_sequence_numbers table and handler to indexer-alt-framework, add as an optional pipeline to indexer-alt

### PR DESCRIPTION
**Pull request by wlmyng**
## Description 

Framework introduces `cp_sequence_numbers` which maps a given checkpoint sequence number to its first tx and containing epoch. This will be used later during pruning, for tables that need more info than the prunable checkpoint range.

## Test plan 

How did you test the new or updated feature?

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

File: crates/sui-indexer-alt-framework/migrations/2024-12-11-234958_cp_mapping/up.sql
Comment (lxfind) [inline crates/sui-indexer-alt-framework/migrations/2024-12-11-234958_cp_mapping/up.sql:1]: checkpoint_mapping is probably not too bad as a full name?

File: crates/sui-indexer-alt-framework/src/schema.rs
Comment (lxfind) [inline crates/sui-indexer-alt-framework/src/schema.rs:26]: Is this generated by the script?

Comment (wlmyng) [reply crates/sui-indexer-alt-framework/src/schema.rs:26]: yes, but i suppose we dont need this

**Review by amnn — COMMENTED** [review]
LGTM, modulo the nits from @lxfind ! Will leave it to Xun to give the final stamp.

**Review by amnn — APPROVED** [review]
💯

**Pull request merged**
